example/cumulus-tf/variables.tf

```diff
@@ -166,7 +166,7 @@ variable "api_gateway_stage" {
 
 variable "api_reserved_concurrency" {
   type = number
-  default = 8
+  default = 15
```

Comment:
If this value is configurable, then why do we need to update our default? DAACs can update this value without us changing it in core.

I suppose the argument is that our default value should be sufficient to serve the dashboard?

Reply:
Yeah actually they should, but most won't configure it, and I got hit with just 8.

Reply:
In that case, should we be updating the default in `tf-modules/cumulus/variables.tf` and possibly removing this?

Reply:
I would think that we still might want it configurable for example deployments, but yeah we definitely want to update the default in `tf-modules/cumulus/variables.tf`

Reply:
Uh yup, didn't remember/know about that. 99bcdc1c74

Reply:
whoops, wrong branch....

Reply:
f82cb293a6